postman.json

```diff
@@ -0,0 +1,265 @@
+{
```

Comment:
It is good, you included this file, so others can easily check out your app :+1: